spark/src/main/scala/org/apache/spark/delta/sharing/PreSignedUrlCache.scala

```diff
@@ -42,7 +42,7 @@ class CachedTable(
     val idToUrl: Map[String, String],
     val refs: Seq[WeakReference[AnyRef]],
     @volatile var lastAccess: Long,
-    val refresher: () => Map[String, String])
+    val refresher: () => (Map[String, String], Long))
```

Comment:
Consider updating the comment about refresh in function header.

Reply:
updated.